package.json

```diff
@@ -65,6 +65,7 @@
   "devDependencies": {
     "@types/jest": "29.5.12",
     "colors": "1.4.0",
+    "ejs": "3.1.9",
```

Comment:
This is necessary for testing the express server's render method (called via `renderAsync` from `PromiseMiddleware`), which wasn't being tested at all previously. I also couldn't find usage within the OpenTable codebase, but this is a public repo so it could be used somewhere in the world.

Reply:
It'd be better if you replace it with a mock and add it as an injection override. This is one of the more annoying dependencies in the system. :)

Reply:
Done